docs/data/stashcache/install-origin.md

```diff
@@ -35,9 +36,8 @@ Before starting the installation process, consider the following points:
 
 As with all OSG software installations, there are some one-time steps to prepare in advance:
 
-* Ensure the host has [a supported operating system](/release/supported_platforms.md).  We strongly recommend
-  all origin servers run RHEL7 or later; this document only covers systemd-based services.  For RHEL6 support,
-  please contact <support@opensciencegrid.org>
+* Ensure the host has [a supported operating system](/release/supported_platforms.md).
+  At this time, we only support RHEL7-based cache servers.
```

Comment:
Copy/paste error?

My preference is to make the origin "strongly prefer" RHEL7 - but require RHEL7 for the cache.

Reply:
I meant to do that but if we're really OK with new origins being RHEL6 I guess I can undo it. Do we have instructions anywhere on how to do it?